.gitignore

```diff
@@ -13,3 +13,5 @@ project/plugins/project/
 .vagrant
 VERSION
 
+# Intellij
```

Comment:
Any IDE/editor-specific go to global gitignore.

Reply:
True.. Thanks